### PR TITLE
Add circuit_breakers to TCP connections

### PR DESCRIPTION
**Pull request by PierreF**
## Description

Circuit breaker were not supported on TCP mapping, making impossible to have more than 1024 connections per ambassador/envoy. This PR add circuit_breakers to TCP mapping (it reuse exactly what is done in HTTP mapping).

More than 1024 TCP connections is important when the TCP service exposed is using persistent connection that are long lived and you expect more than 1024 client.

## Related Issues
It's the PR for #2193

## Testing

A testing step present in #2193 still apply.

## Tasks That Must Be Done
- [x] Did you update CHANGELOG.md?
- [ ] Did you add or update tests?
- [x] Did you update documentation?
- [ ] Were there any special dev tricks you had to use to work on this code efficiently?
  + [ ] Did you add them to DEVELOPING.md?
- [ ] Is this a build change?
  + [ ] If so, did you test on both Mac and Linux?

### CODE REVIEW

**Comment by kflynn** [comment]
Also, are you on our [Slack](https://d6e.co/slack)? That might be a good place to talk further about this. 🙂

**Comment by PierreF** [comment]
Fixed your comment and I just joinned Slack.

File: python/ambassador/ir/irtcpmapping.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
 from .irbasemapping import IRBaseMapping
 from .irbasemappinggroup import IRBaseMappingGroup
+from .irhttpmapping import IRHTTPMapping
```
Comment (kflynn) [inline python/ambassador/ir/irtcpmapping.py:8]: This is unneeded now, right?

**Comment by PierreF** [comment]
Added test (that works :)) for TCP mapping & circuit breaker.

**Comment by kflynn** [comment]
Looks really good in general! Thanks in advance for helping clean stuff up for us. 🙂

**Pull request merged**